rebar.config

```diff
@@ -2,10 +2,10 @@
 %% -*- tab-width: 4;erlang-indent-level: 4;indent-tabs-mode: nil -*-
 %% ex: ts=4 sw=4 ft=erlang et
 {deps, [
-        {ibrowse, ".*",
-         %% Pin here, becase 555f707 (pr #155) introduces an ipv6 bug we've not fixed
-         {git, "https://github.com/cmullaparthi/ibrowse.git", {ref, "c97136cfb61fcc6f39d4e7da47372a64f7fca04e"}}},
-        {envy, {git, "https://github.com/markan/envy.git", {branch, "master"}}}
+        %% Pin here, becase 555f707 (pr #155) introduces an ipv6 bug we've not fixed
+        {ibrowse, ".*", {git, "https://github.com/cmullaparthi/ibrowse.git", {ref,    "c97136cfb61fcc6f39d4e7da47372a64f7fca04e"}}},
```

Comment:
Is ibrowse needed any longer? I don't think it should be.

Reply:
ibrowse deleted.
https://buildkite.com/chef-oss/chef-mini-s3-master-verify/builds/8
https://buildkite.com/chef/chef-chef-server-master-verify/builds/2436#b9944907-8fe2-4dd2-94c3-9e3dbb97278c
https://buildkite.com/chef/chef-chef-server-master-omnibus-adhoc/builds/1666#a062a6e9-23e2-4239-a804-e46ab54c2872